src/blindrsa.ts

```diff
@@ -13,40 +15,41 @@ import {
     rsavp1,
 } from './util.js';
 
-import sjcl from './sjcl/index.js';
-
 export enum PrepareType {
     Deterministic = 0,
     Randomized = 32,
 }
 
 export type BlindOutput = { blindedMsg: Uint8Array; inv: Uint8Array };
 
+export interface BlindRSAParams {
+    name: string;
```

Comment:
`name` does not appear to be used. Is it meant for debugging? In this case, either mark it as optional or remove it

Reply:
there is a top-level function `getSuiteByName` which uses the `name` field.